stdlib/public/BackDeployConcurrency/AsyncIteratorProtocol.swift

```diff
@@ -56,10 +57,10 @@ import Swift
 ///
 /// At the call site, this looks like:
 ///
-///     for await i in Counter(howHigh: 10) {
-///       print(i, terminator: " ")
+///     for await number in Counter(howHigh: 10) {
+///       print(number, terminator: " ")
```

Comment:
I renamed `i` to `number` just like the other examples.

Reply:
That looks like a good change to me. It's consistent, and "number" means more to the reader than "i".